doc/WebSite/Zero/Permission-Management.md

```diff
@@ -43,23 +43,23 @@ a role in one method call:
         }
     }
 
-In this example, we get a **RoleId** and list of granted permission
-names (input.GrantedPermissionNames is a List&lt;string&gt;) as input.
-We use **IPermissionManager** to find all **Permission** objects by
-name. Then we call **SetGrantedPermissionsAsync** method to update
-permissions of the role.
+In this example, we get a **RoleId** and a list of granted permission
+names (input.GrantedPermissionNames is a List&lt;string&gt;) as an input.
+We then use the **IPermissionManager** to find all the **Permission** objects by
+name. After that, we call the **SetGrantedPermissionsAsync** method to update
+the permissions of the role.
 
 There are also other methods like GrantPermissionAsync and
-ProhibitPermissionAsync to control permissions one by one.
+ProhibitPermissionAsync to control the permissions one-by-one.
 
 ### User Permissions
 
-While role-based permission management can be enough for most
-applications, we may need to control permissions per user. When we
-define a permission setting for a user, it overrides permission setting
-comes from roles of the user.
+While the role-based permission management can be enough for most
+applications, we may need to control the permissions per user. When we
+define a permission setting for a user, it overrides the permission setting
+coming from the roles of the user.
 
-As an example; Assume that we have an application service to prohibit a
+As an example; Imagine that we have an application service that prohibits a
```

Comment:
"As an example`,` `i`magine that we have an application service `method for prohibiting` a permission"